util/usernames

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
```

Comment:
Should this file have a `.py` extension, or does a hashbang indicate that it should not?